app/screens/selector_screen/selector_screen.tsx

```diff
@@ -131,35 +135,47 @@ export default class SelectorScreen extends PureComponent<Props, State> {
 
         switch (this.props.dataSource) {
         case ViewTypes.DATA_SOURCE_USERS: {
-            const currentList = this.state.multiselectSelected as UserProfile[];
+            const currentList = this.state.multiselectSelected as Dictionary<UserProfile>;
```

Comment:
Should this be called `currentMap` now? Or something other than list, maybe `currentSelected`